examples/dt/uni04delta/control-plane/service-values.yaml

```diff
@@ -52,26 +52,20 @@ data:
 
   manila:
     enabled: true
-    manilaShares:
-      share1:
-        customServiceConfig: |
-          [DEFAULT]
-          enabled_share_backends = cephfs
-          enabled_share_protocols = cephfs
+    customServiceConfig: |
```

Comment:
Instead of having this only on `share1`, we make it global so we get the API config for free.

Reply:
So the generated `edpm-nodeset.yaml` described in:

https://github.com/openstack-k8s-operators/architecture/blob/main/examples/dt/uni04delta/edpm.md#crs-nodeset

will change to this after this patch merges

```diff
--- edpm-nodeset.yaml	2024-06-07 11:49:49.647545091 -0400
+++ /tmp/edpm-nodeset.yaml	2024-06-07 11:49:40.726491002 -0400
@@ -338,6 +338,20 @@
         haproxy.router.openshift.io/timeout: 60s
     enabled: true
     template:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_share_backends = cephfs
+        enabled_share_protocols = cephfs
+
+        [cephfs]
+        driver_handles_share_servers = False
+        share_backend_name = cephfs
+        share_driver = manila.share.drivers.cephfs.driver.CephFSDriver
+        cephfs_conf_path = /etc/ceph/ceph.conf
+        cephfs_cluster_name = ceph
+        cephfs_auth_id = openstack
+        cephfs_volume_mode = 0755
+        cephfs_protocol_helper_type = CEPHFS
       manilaAPI:
         networkAttachments:
         - internalapi
@@ -356,20 +370,6 @@
         replicas: 1
       manilaShares:
         share1:
-          customServiceConfig: |
-            [DEFAULT]
-            enabled_share_backends = cephfs
-            enabled_share_protocols = cephfs
-
-            [cephfs]
-            driver_handles_share_servers = False
-            share_backend_name = cephfs
-            share_driver = manila.share.drivers.cephfs.driver.CephFSDriver
-            cephfs_conf_path = /etc/ceph/ceph.conf
-            cephfs_cluster_name = ceph
-            cephfs_auth_id = openstack
-            cephfs_volume_mode = 0755
-            cephfs_protocol_helper_type = CEPHFS
           networkAttachments:
           - storage
           replicas: 1

```